tests/asgi/test_ws.py

```diff
@@ -1118,6 +1118,9 @@ async def on_websocket(self, req, ws):
     async with conductor as c:
         if accept:
             async with c.simulate_ws() as ws:
+                # Make sure the responder has a chance to reach the raise point
+                for _ in range(3):
+                    await asyncio.sleep(0)
```

Comment:
Since we have same code in two places maybe we could extract it to some function e.g. "wait"?